docs/source/basic_examples.rst

```diff
@@ -1022,6 +1022,23 @@ useful key for metal-containing molecules. You can use the
     stk.MoleculeMongoDb = _old_molecule_init
     pymongo.MongoClient = _mongo_client
 
+Accessing Vertex classes
+========================
+
+The vertex classes that make up toplogy graphs in :mod:`stk` can be
+accessed if required.
+
+.. testcode:: accessing-vertex-classes
+
+    import stk
+
+    base_vertex = stk.Vertex(0, [0, 0, 0])
```

Comment:
I think the example should demonstrate to the user how to solve a specific problem. So in this case, maybe like make a new cage topology graph? 

I thiink giving an example of making a `stk.Vertex` instance is bad, because you actually never want to make an instance of it -> it has unimplemetned methods. You only use it for the purposes inheritence, when defining a new Vertex class.